tests/functional/testMultihop.js

```diff
@@ -0,0 +1,125 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ const assert = require('assert');
+ const vpn = require('./helper.js');
+ 
+ describe('Server list', function() {
+   let servers;
+   let currentCountryCode;
+   let currentCity;
+ 
+   this.timeout(240000);
+   this.ctx.authenticationNeeded = true;
+ 
+   beforeEach(async () => {
+     await vpn.waitForElementAndClick('serverListButton');
+     await vpn.wait();
+ 
+     servers = await vpn.servers();
+     currentCountryCode = await vpn.getSetting('current-server-country-code');
+     currentCity = await vpn.getSetting('current-server-city');
+ 
+     for (let server of servers) {
+       if (currentCountryCode === server.code) {
+         for (let city of server.cities) {
+           if (city.name == currentCity) {
+             currentCity = city.localizedName;
+             break;
+           }
+         }
+       }
+     }
+     console.log(
+         'Current city (localized):', currentCity,
+         '| Current country code:', currentCountryCode);
+   });
+
+   it('Checking multihop functionality', async () => {
+    for(let i=0; i<servers.length; i++){
+      // wait for select entry      
+      await vpn.waitForElementAndClick('multiHopSelector/tabMultiHop');
+      await vpn.wait()    
+       
+      // select entry
+      await vpn.waitForElementAndClick('buttonSelectEntry');    
+      
+      // wait for country of choice to be visible    
+      const server = servers[i]
+      const countryId = 'serverCountryList/serverCountry-' + server.code;
+      const exitCountryId = 'serverCountryList/serverCountry-' + servers[0].code;      
```

Comment:
you  always pick the same exit node?

Reply:
it'll need to be updated once bug is fixed